Automattic/woocommerce-payments
# Add Tracks to Place Order button

#### Changes proposed in this Pull Request

Record Tracks event when clicking the Place Order button when using WooPayments.









#### Testing instructions





1. Add a product to the cart.
2. Go to the classic checkout page.
3. Make sure WooPayments is selected in Payment Methods
4. Click on the Place Order button
5. In ~5 minutes, check MC live view for a `wcpay_checkout_place_order_button_click` event from your site.
6. Select a different gateway (Stripe, Check Payments"
7. Click on the Place Order Button
8. No event should be recorded.
9. Repeat the above steps with Blocks checkout.

-------------------


**Post merge**
